src/sidebar/util/sentry.js

```diff
@@ -54,21 +54,10 @@ export function init(config) {
 
     // Ignore various errors due to circumstances outside of our control.
     ignoreErrors: [
-      // Ignore transient network request failures. Some of these ought to be
+      // Ignore network request failures. Some of these ought to be
       // caught and handled better but for now we are suppressing them to
       // improve the signal-to-noise ratio.
-      'Failed to fetch', // Chrome
-      'Fetch operation failed',
-      'NetworkError when attempting to fetch resource', // Firefox
-
-      // Ignore network request failures due to empty JSON bodies.
-      // TODO - Consider removing these once `fetch` callers have been changed
-      // to handle `Response.json()` calls failing.
-      'JSON.parse: unexpected end of data', // Firefox
-      'Unexpected end of JSON input', // Opera Mobile
-
-      // Ignore network request cancellations
-      'AbortError: The operation was aborted.', // Firefox
+      'Network request failed', // Standard message prefix for `FetchError` errors
```

Comment:
Here we see the benefit of standardizing the format of network errors. We could also potentially do an `instanceof FetchError` check in this module and customize the reporting logic.